src/app/action-items/action-items.component.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { DaysLeftToDeadlineService } from './../services/daysLeftToDeadlineService/days-left-to-deadline.service';
+import { async, ComponentFixture, TestBed, tick, fakeAsync } from '@angular/core/testing';
+import { of } from 'rxjs';
+import { TasksService, ActionTasksElement } from '../services/tasksService/tasks.service';
+import { ActionItemsComponent, ActionTasksElementMapped } from './action-items.component';
+import { CircleColorPipe } from '../pipes/circleColorPipe/circle-color.pipe';
+
+describe('ActionItemsComponent', () => {
+  let component: ActionItemsComponent;
+  let fixture: ComponentFixture<ActionItemsComponent>;
+  let tasksService: TasksService;
+  let daysLeftToDeadlineService: DaysLeftToDeadlineService;
+  let element;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ActionItemsComponent, CircleColorPipe],
+      providers: [TasksService]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    tasksService = TestBed.get(TasksService);
+    daysLeftToDeadlineService = TestBed.get(DaysLeftToDeadlineService);
+    fixture = TestBed.createComponent(ActionItemsComponent);
+    component = fixture.componentInstance;
+    element = fixture.nativeElement;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should have the initial value of loading=true;', () => {
+    expect(component.loading).toBe(true);
+  });
+
+  it('should stop loading after 1s ', () => {
```

Comment:
Please fix test name - component shouldn't know anything about implementation details of tasksService.